modules/apps/object/object-rest-test/src/testFunctional/macros/OAuth2ForObject.macro

```diff
@@ -0,0 +1,55 @@
+definition {
+
+	macro addAuth2Application {
```

Comment:
Can we change the name to addOAuth2Application? This way we stay aligned with OAuth2 we use after :)

Reply:
Updated, thanks